mozci/task.py

```diff
@@ -225,7 +226,7 @@ def __post_init__(self):
 
     @property
     def classifications(self):
-        return set(t.classification for t in self.tasks if t.failed)
+        return [(t.classification, t.classification_note) for t in self.tasks if t.failed]
```

Comment:
Looks like we could move `def classifications` up to the `RunnableSummary` class.

Reply:
Makes sense, I'll do that.

Reply:
Done with dac0242.